torch/_inductor/async_compile.py

```diff
@@ -357,3 +358,10 @@ def wait(self, scope: Dict[str, Any]) -> None:
     pass
 else:
     AsyncCompile.warm_pool()
+
```

Comment:
So was this just missing before?

Reply:
I think so. It only happened sometimes (when the child exited uncleanly?) so not normally seen.